src/Identity/Identity.slnf

```diff
@@ -86,7 +86,9 @@
       "src\\Mvc\\Mvc.Testing\\src\\Microsoft.AspNetCore.Mvc.Testing.csproj",
       "src\\ObjectPool\\src\\Microsoft.Extensions.ObjectPool.csproj",
       "src\\Http\\Metadata\\src\\Microsoft.AspNetCore.Metadata.csproj",
-      "src\\Localization\\Abstractions\\src\\Microsoft.Extensions.Localization.Abstractions.csproj"
+      "src\\Localization\\Abstractions\\src\\Microsoft.Extensions.Localization.Abstractions.csproj",
+      "src\\Components\\Web\\src\\Microsoft.AspNetCore.Components.Web.csproj",
```

Comment:
Did you actually need the components project when working with identity here?  I think the other two additions are fine, but I'd rather not load components in the Identity slnf unless its really needed

Reply:
Yes, it was needed to be referenced via the test project so that I could run the tests in Visual Studio, otherwise the test compilation failed.